test/migrators/division/multiplication.hrx

```diff
@@ -0,0 +1,25 @@
+<==> input/entrypoint.scss
+$a: 2000px / 2;
+$b: $a / 4;
+$c: $a / 5;
+$d: ($a / 8);
+$e: ($a / 10);
+$f: ($a / 20);
+$g: $a/40;
+$h: $a/50;
+$i: $a/80;
+$j: ($a/100);
+$k: ($a/1000);
```

Comment:
Test a division operation where the dividend is itself a mathematical operation—you'll need to be sure to keep it parenthesized in that case.